sale_datetime_confirm/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#########################################################################
+#                                                                       #
+# Copyright (C) 2016  Akretion                                          #
+#                                                                       #
+# This program is free software: you can redistribute it and/or modify  #
+# it under the terms of the GNU Affero General Public License as        #
+# published by the Free Software Foundation, either version 3 of the    #
+# License, or (at your option) any later version.                       #
+#                                                                       #
+# This program is distributed in the hope that it will be useful,       #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+# GNU Affero General Public Licensefor more details.                    #
+#                                                                       #
+# You should have received a copy of the                                #
+# GNU Affero General Public License                                     #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. #
+#                                                                       #
+#########################################################################
```

Comment:
@florian-dacosta   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code.
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3